content/hardware/02.hero/boards/uno-r4-wifi/tutorials/touch/content.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Uno R4 Capacitor Tutorial
+description: Learn to use the built-in capacitive sensing capabilities of the Arduino® Uno R4.
+tags: [Arduino, Capacitive Sensing, Uno R4]
+author: Pedro Lima
+---
+
+Capacitive sensing is a technology that detects changes in capacitance to determine the presence or absence of a conductive object, such as a human finger. This principle is widely used in touch-sensitive devices. The Arduino® Uno R4, both the [WiFi](https://store.arduino.cc/products/arduino-uno-r4-wifi) and [Minima](https://store.arduino.cc/products/arduino-uno-r4-minima) versions, come equipped with built-in capacitive sensing capabilities, making it easier to integrate touch inputs into your projects.
+
+![Sensor Example](assets/Touch_Cover_001.gif)
```

Comment:
I think we could add this as a duplicate after the example section, so it works as a nice visual addition of what to expect or how it would work. The image placement here is good because it shows the reader from the introduction what they are expecting to learn within the tutorial, so I think it would be nice to copy this line and add it next to the example section.